src/zeekygen/IdentifierInfo.cc

```diff
@@ -35,8 +35,17 @@ IdentifierInfo::~IdentifierInfo() {
 }
 
 void IdentifierInfo::AddRedef(const string& script, zeek::detail::InitClass ic, zeek::detail::ExprPtr init_expr,
-                              const vector<string>& comments) {
-    Redefinition* redef = new Redefinition(script, ic, std::move(init_expr), comments);
+                              vector<string> comments) {
+    bool omit_value = false;
+    for ( auto it = comments.begin(); it != comments.end(); ++it ) {
+        if ( it->find("@docs-omit-value") != std::string::npos ) {
+            it = comments.erase(it);
```

Comment:
Doesn't really matter due to the `break`, but the `++it` in the for head wouldn't work with `it = comments.erase(it)` as that would jump one element, maybe just remove the `it` assignment?